apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/StoreMetrics.scala

```diff
@@ -36,6 +36,34 @@ class StoreMetrics(metricsFactory: LabeledMetricsFactory)(metricsContext: Metric
       0L,
     )(metricsContext)
 
+  val lastIngestedOffset: Gauge[Long] = metricsFactory.gauge(
```

Comment:
Any particular reason we care about the offset? Or could it also be the last ingested record time? The latter is a bit easier to correlate across different network components (offset is participant local), and can be compared to other time sources to compute delays.